DiningStack/expandedItems.json

```diff
@@ -0,0 +1,1041 @@
+{
+  "eateries": [
+    {
+      "name": "Atrium Café",
+      "slug": "Atrium-Cafe",
+      "categories": [
+        {
+          "category": "Main",
+          "stations": [
+            {
+              "station": "Hot Fresh Pizza",
+              "diningItems": [
+                {
+                  "item": "Cheese Pizza",
+                  "options": [],
+                  "price": "$4.49"
+                },
+                {
+                  "item": "Veggie Pizza",
+                  "options": [],
+                  "price": "$4.99"
+                },
+                {
+                  "item": "Pepperoni Pizza",
+                  "options": [],
+                  "price": "$4.99"
+                },
+                {
+                  "item": "Gourmet Pizza",
+                  "options": [],
+                  "price": "$5.69"
+                }
+              ]
+            },
+            {
+              "station": "Specialty Sandwiches",
+              "diningItems": [
+                {
+                  "item": "Joint Ventures",
+                  "options": [],
+                  "price": "$6.99"
+                },
+                {
+                  "item": "Parker Center Pesto",
+                  "options": [],
+                  "price": "$6.99"
+                },
+                {
+                  "item": "PhD Eggplant",
+                  "options": [],
+                  "price": "$6.99"
+                },
+                {
+                  "item": "Financial Derivatives",
+                  "options": [],
+                  "price": "$6.99"
+                },
+                {
+                  "item": "Immersion Chicken",
+                  "options": [],
+                  "price": "$6.99"
+                },
+                {
+                  "item": "Merger of Equals",
+                  "options": [],
+                  "price": "$6.99"
+                },
+                {
+                  "item": "Supply Chain Wraps",
+                  "options": [],
+                  "price": "$6.99"
+                }
+              ]
+            },
+            {
+              "station": "Custom Deli Sandwiches",
+              "diningItems": [
+                {
+                  "item": "Gourmet Sandwiches",
+                  "category": "",
+                  "options": [
+                    {
+                      "label": "Bread",
+                      "options": [
```

Comment:
might not be ideal to have two fields named `options`, we should rename one

Reply:
Renamed